### ISSUE TITLE
StopPropagation for editor

### ISSUE THREAD

**Issue by KES777** [olifolkerd/tabulator#1666]
**Describe the bug**
When we are editing cell with "autocomplete" editor and use 'HOME' / 'END'
instead of cursor is moved to the end of edited string, the table is scrolled to the top/bottom

**Tabulator Info**
- Which version of Tabulator are you using? 4.1.3
- Post a copy of your construct object if possible so we can see how your table is setup

```javascript
var tabledata = [
	{id:1, country_id: 1 },
        .... x40 lines
	{id:10, country_id: 1 },
	];
  
var country_list = {
  1: "Germany",
  2: "Ukraine",
  3: "Canada",
};

//Build Tabulator
var table = new Tabulator("#example-table", {
    height:"311px",
    data: tabledata,
    columns:[
        {title:"ID", field:"id", width:40 },
        {title:"Country", field:"country_id", width:130, editor:"autocomplete", editorParams:{values: country_list, showListOnEmpty: true, }, formatter: "lookup", formatterParams: country_list, },       
    ],
});
```

**To Reproduce**
[JS fiddle](https://jsfiddle.net/KES777/Lykuaq7j/2/)
1. Start to edit 'Country' field
2. Move cursor to the start of string
3. Press 'End' key
4. Table is scrolled to the bottom

**Expected behavior**
Cursor should be moved to the end of the string

**Screenshots**
![__](https://user-images.githubusercontent.com/799021/50148442-2ee1bb80-02c1-11e9-8fc4-b96556ceb5d9.jpg)
![__2](https://user-images.githubusercontent.com/799021/50148444-30ab7f00-02c1-11e9-9f92-1de5aeea257c.jpg)
1. We editing first 'Country' cell. Cursor is at beginning of string value
2. We are on top of table
3. We press 'End' key. List still is displayed
4. We are on bottom of the table (but just cursor should be moved to the end of string at editor)

**Desktop (please complete the following information):**
 - OS: Linux Mint 18.3
 - Browser FF
 - Version 64

**Comment by olifolkerd** [comment]
Hey @KES777 

That is a very good point, i can see how that would be annoying, i will add that to the roadmap for the 4.2 release.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
A fix for this has been pushed to the 4.2 branch which will be released on monday.

Cheers

Oli :)

**Comment by KES777** [comment]
works fine with [4.2.1](https://jsfiddle.net/KES777/Lykuaq7j/2/) version
